File: server/Models/Photo.cs
namespace reactNet.Models;
public class Photo
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public string Img { get; set; }
    public Account Creator { get; set; }
    public string CreatorId { get; set; }

}

File: client/src/models/Photo.js
export class Photo{
    constructor(data){
        this.id = data.id
        this.title = data.title
        this.description = data.description
        this.img = data.img
        this.createdAt = new Date(data.createdAt).toLocaleDateString()
        this.updatedAt = new Date(data.updatedAt).toLocaleTimeString()
        this.creator = data.creator
        this.creatorId = data.creatorId
    }
}